android/build.gradle

```diff
@@ -39,22 +35,12 @@ tasks.withType(JavaCompile) {
 android.libraryVariants.all { variant ->
     def name = variant.buildType.name
 
-    // For each variant add a task to copy the core classes so they are included in the AAR.
-    def copyTask = project.tasks.create(name: "copyCoreToAndroid${name.capitalize()}", type: Copy) {
-        from(project(':core').sourceSets.main.output)
-        into(variant.javaCompile.destinationDir)
-    }
-    copyTask.dependsOn variant.javaCompile
-    variant.outputs.each { output ->
-        output.packageLibrary.dependsOn copyTask
-    }
-
     // Also define a task to create a raw JAR to support non-AAR users.
     def rawJarTask = project.tasks.create(name: "rawAndroidJar${name.capitalize()}", type: Jar) {
         archiveName="keen-client-api-android-${project.version}.jar"
         from(variant.javaCompile.destinationDir)
     }
-    rawJarTask.dependsOn copyTask
+    rawJarTask.dependsOn javaCompile
```

Comment:
Yes, basically. I'll need to test this part to make sure it behaves as expected though. Looking now, I think I may actually want `variant.javaCompile` instead of just `javaCompile`, but they may implicitly mean the same thing in this context. Anyway testing should confirm.